### DISCUSSION TITLE
Expose AxiosHeaders

### DISCUSSION THREAD

**Discussion by kuizuo** [axios/axios#5160] [General]
In `lib\axios.js` file. `AxiosHeaders` are not exported.

![image](https://user-images.githubusercontent.com/61005888/196755739-7cd41e28-c2f2-43dd-8ef0-ccc3569269a3.png)

But `AxiosHeaders` is very useful for me, so why not expose it? Just like the one above.

**Comment by jasonsaayman** [comment]
we only export certain functionality to guard against using our utils etc. please check the package.json and if this is not exported then we probably will not be updating that export list
